CHANGES.rst

```diff
@@ -5,7 +5,9 @@ CHANGES
 3.10.1 (unreleased)
```

Comment:
Should become 3.11 or maybe even 4.0 release.

Reply:
done